pkg/restic/common.go

```diff
@@ -347,3 +353,27 @@ func CmdEnv(backupLocation *velerov1api.BackupStorageLocation, credentialFileSto
 
 	return env, nil
 }
+
+// GetInsecureSkipTLSVerifyFromBSLForRestic get insecureSkipTLSVerify flag from BSL configuration,
+// Then return --insecure-tls flag with boolean value as result.
+func GetInsecureSkipTLSVerifyFromBSLForRestic(backupLocation *velerov1api.BackupStorageLocation, logger logrus.FieldLogger) string {
```

Comment:
since it's not supposed to be used outside of the `restic` package we don't need to export it?
Same reason we don't need to call it `xxxxForRestic`

Reply:
Creating a new function could help to keep the caller's complexity not growing.
Agree with the conclusions above. How about I put the function into repossitory_manager.go, rename it, and don't export the function?